src/webview/leetCodePreviewProvider.ts

```diff
@@ -49,21 +59,35 @@ class LeetCodePreviewProvider implements Disposable {
         }
     }
 
-    public async provideHtmlContent(node: IProblem): Promise<string> {
-        return await this.renderHTML(node);
+    private parseDescription(descString: string, problem: IProblem): IDescription {
+        const [
+            /* title */, ,
+            url, ,
```

Comment:
Each comma without variable means ignore this line (normally empty line, or something not needed)